test/system/CLI.js

```diff
@@ -0,0 +1,74 @@
+import tap from 'tap';
```

Comment:
Personal preference: It can be a hassle to have UPPERCASE filenames in git, so I tend to stay away from uppercase letters. It becomes super annoying when renaming them in the future because not all file systems differentiate between uppercase and lowercase letters. In short: I'd call this file `test/system/cli.js` instead.